conan/internal/api/new/qbs_lib.py

```diff
@@ -0,0 +1,103 @@
+from conan.internal.api.new.cmake_lib import source_cpp, source_h, test_main
+
+
+conanfile_sources = '''
+import os
+
+from conan import ConanFile
+from conan.tools.qbs import Qbs
+
+
+class {{package_name}}Recipe(ConanFile):
+    name = "{{name}}"
+    version = "{{version}}"
+
+    exports_sources = "*.cpp", "*.h", "*.qbs"
+    settings = "os", "compiler", "arch"
+    options = {"shared": [True, False]}
+    default_options = {"shared": False}
+
+    def build(self):
+        qbs = Qbs(self)
+        qbs.profile = ""
+        qbs_config = {"products.{{name}}.isShared": "true" if self.options.shared else "false"}
+        qbs.add_configuration("default", qbs_config)
+        qbs.build()
+
+    def package(self):
+        qbs = Qbs(self)
+        qbs.profile = ""
+        qbs.install()
+
+    def package_info(self):
+        self.cpp_info.libs = ["{{name}}"]
+'''
+
+qbs_lib_file = '''
+    Library {
+        property bool isShared: true
+        name: "{{name}}"
+        type: isShared ? "dynamiclibrary" : "staticlibrary"
+        files: [ "{{name}}.cpp" ]
+        Group {
+            name: "headers"
+            files: [ "{{name}}.h" ]
+            qbs.install: true
+            qbs.installDir: "include"
+        }
+        Depends { name: "cpp" }
+        Depends { name: "bundle" }
+        bundle.isBundle: false
+        install: true
+        qbs.installPrefix: ""
+    }
+'''
+
+test_conanfile_v2 = """import os
+
+from conan import ConanFile
+from conan.tools.build import can_run
+from conan.tools.qbs import Qbs
+from conan.tools.build import cmd_args_to_string
+
+class {{package_name}}TestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "PkgConfigDeps"
```

Comment:
I guess we want to move this later to ``QbsDeps``, isn't it? later PR too.

Reply:
I was thinking about it, but that creates a chicken-egg problem in tests - we will test QbsDeps using QbsDeps, is it OK?
I think, pkgconfig is fine here - it works fine in simple cases like this. Also, we don't invoke the pkg-config binary from Qbs (we simply read pc files and create modules, Qbs handles the rest). So no external deps here.

Reply:
The ``conan new`` templates aim to generate the "canonical" and most recommended practice. If the ``QbsDeps`` is the generator that you prefer users to use, then it should be in the ``conan new`` template, and documented as such.

It is fine to test ``QbsDeps`` with the ``conan new`` template, as it is actually the closest to the user flow, so very recommended to test it with it.